### PR TITLE
Added additional attributes for Cronjob and Node entities : Fixes issue and #1509

### LINKED ISSUE #1509: Add important missing metrics for Cronjob and Node
We are not getting important metrics `failedJobsHistoryLimit` and `successfulJobsHistoryLimit` for cronjobs in kube-state-metrics.

Similarly `systemUUID` for node is not being reported.

I have checked the https://github.com/kubernetes/kube-state-metrics/blob/master/docs/cronjob-metrics.md and https://github.com/kubernetes/kube-state-metrics/blob/master/docs/node-metrics.md doc page and these important metrics are not being reported.

We need the above information for monitoring the k8s cluster using kube-state-metrics.

### PR DESCRIPTION
**Pull request by sumitk23**
**What this PR does / why we need it**: 
This PR adds some new attributes for Cronjob and Node resources.
For Cronjob, added support for attributes : successfulJobHistoryLimit and FailedJobHistoryLimit
For Node, added support for attribute : SystemUUID

**How does this change affect the cardinality of KSM**:  No change

**Which issue(s) this PR fixes** 
Fixes #1509

### CODE REVIEW

**Comment by sumitk23** [comment]
Hi @mrueg @brancz Can you pls take a look at this PR and provide your comments.

**Comment by sumitk23** [comment]
Hi @mrueg @brancz Please take a look at this PR and provide your comments.

**Review by mrueg — APPROVED** [review]

**Comment by mrueg** [comment]
In general this change looks good to me, waiting for other maintainers to provide a review as well.

File: docs/cronjob-metrics.md
Comment (dgrisonnet) [inline docs/cronjob-metrics.md:15]: Since these are new metrics, they should be EXPERIMENTAL for now

Comment (sumitk23) [reply docs/cronjob-metrics.md]: @dgrisonnet I have changed these new metrics to EXPERIMENTAL, pls re-review and approve.

**Comment by dgrisonnet** [comment]
LGTM, thank you @sumitk23!

/lgtm

**Pull request merged**
